Games/types/Mafia/roles/cards/GunFramer.js

```diff
@@ -0,0 +1,26 @@
+const Card = require("../../Card");
+const { PRIORITY_GUN_FRAME } = require("../../const/Priority");
+
+module.exports = class GunFramer extends Card {
```

Comment:
This seems like a one-off meeting? If you want to give illusionist the power to continually frame people (e.g. when they're gunned by Arms Dealer), you need to add some more mechanics~

Reply:
ok~ I was referring to how the Illusionist can get more guns from other sources.
The idea is that "For any gun Illusionist receives, it can mask the shooter name with any of their choice".

The starting item of illusionist is actually a "Gun" with a shooter property set to the illusionist.

The illusionist can get a gun from an arms dealer, and still be able to "mask the shooter" for the arms dealer's new gun.